app/models/sac_cas/role/mitglied_uniqueness_validation.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacCas::Role::MitgliedUniquenessValidation
+  extend ActiveSupport::Concern
+
+  included do
+    validates :type, uniqueness: { scope: [:person_id, :group_id] }
```

Comment:
Ich glaube das entspricht noch nicht dem gewünschten Verhalten.
Die `Mitglied` Rolle wird ja immer in der Gruppe `Mitglieder` erstellt, die `Neuanmeldung` Rolle immer in einer Gruppe `SektionsNeuanmeldungSektion` oder `SektionsNeuanmeldungNv`.
Nach meinem Verständnis darf es nicht gleichztigig eine `Mitglied` Rolle sowie eine `Neuanmeldung` Rolle innerhalb derselben Sektion geben, wobei die Sektion die parent group von der role group ist.

Reply:
Das feature ist implementiert wie es spezifiziert ist.
Meine Anmerkungen sind durch das ticket #145 abgedeckt.
Also alles i.O. :+1: